pylsp_ruff/plugin.py

```diff
@@ -98,6 +98,8 @@ def pylsp_settings():
             "mccabe": {"enabled": False},
             "pycodestyle": {"enabled": False},
             "pyls_isort": {"enabled": False},
+            "yapf": {"enabled": False},
+            "autopep8": {"enabled": False},
```

Comment:
Shouldn't the Black plugin be included here?

Reply:
Black is a 3rd-party plugin (just like this ruff plugin), so I'd argue that if you manually install it together with ruff, you are asking for it :-P

Reply:
(autopep8 and yapf are included in pylsp, and are enabled by default without user intervention)

Reply:
Let me know if you want me to disable `black` as well :-) 

I don't think we should touch 3rd-party plugins, simply because they are 3rd-party. I'd honestly rather remove `isort` from the default disabled list, as it doesn't come from pylsp-core

Reply:
`pyls_isort` is also the wrong setting? According to the plugin, it should just be:
```python
"isort": {"enabled": False}
```

ref: https://github.com/chantera/python-lsp-isort

(and no `pyls_isort` exists in the pylsp-core: https://github.com/python-lsp/python-lsp-server/blob/develop/pylsp/config/schema.json)

Reply:
> Let me know if you want me to disable black as well :-)

That's up to @jhossbach, but I agree with you and it's better not to touch it.

> I don't think we should touch 3rd-party plugins, simply because they are 3rd-party. I'd honestly rather remove isort from the default disabled list, as it doesn't come from pylsp-core

Sure, that's a good point.

> pyls_isort is also the wrong setting?

It probably is.

Reply:
I agree to remove `isort` altogether from this list and keep black out as well.

Reply:
Fixed